doc/stdlib/math.rst

```diff
@@ -164,11 +172,29 @@ Mathematical Operators
 
    The floored quotient and modulus after division. Equivalent to ``(fld(x,y), mod(x,y))``\ .
 
+.. function:: fld1(x,m)
```

Comment:
Just to be clear, I'm talking about the exact line I'm commenting on `.. function:: fld1(x,m)`.